src/main/java/edu/ie3/datamodel/models/value/WeatherValue.java

```diff
@@ -27,44 +28,66 @@ public class WeatherValue implements Value {
   /** Wind values for this coordinate */
   private final WindValue wind;
 
+  /** Ground temperature value at 0cm depth for this coordinate */
```

Comment:
From point of variable naming, I would now assume that these are no longer linked with a specific depth, right?